target/linux/layerscape/armv8_32b/target.mk

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (C) 2016 Jiang Yutang <jiangyutang1978@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+ARCH:=arm
+BOARDNAME:=ARMv8 32-bit based boards
+CPU_TYPE:=cortex-a9
```

Comment:
Layerscape CPU is cortex A53 or cortex A72 now, but we want it to run on 32-bit mode which is compatible with ARMv7.
So we set CPU_TYPE as cortex-a9 to select 32b toolchain.